packages/manager/apps/dedicated-servers/src/hooks/useCluster.tsx

```diff
@@ -0,0 +1,13 @@
+import { useQuery } from '@tanstack/react-query';
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { getClusters, getClusterQueryKey } from '@/data/api/cluster';
+
+export const useCluster = () => {
+  return useQuery<string[], ApiError>({
+    queryKey: getClusterQueryKey,
+    queryFn: async () => {
+      const { data } = await getClusters();
+      return data;
+    },
+  });
+};
```

Comment:
why do you return only data ?
You may need to know if it's loading, there is an error etc.